### ISSUE TITLE
[Colors] Fix color naming scheme

### ISSUE THREAD

**Issue by zdlm** [Tradeshift/tradeshift-ui#67]
colors
    search in the code for colors used [RGB]
    search in the code for rogue colors [LZA]
    search in code for fade() colors [JMO]

```
come up with the scale of colors [GPA]
figure out new naming scheme
```

https://github.com/Tradeshift/Client-Runtime/issues/523

**Comment by bomouridsen** [comment]
Have starting working on a new color palette. Will define a prober naming scheme at the same time. Inspiration: https://polaris.shopify.com/visuals/colors#color-palette

**Comment by lloydhumphreys** [comment]
@mihneaz please come up with a color range naming scheme?

**Comment by lloydhumphreys** [comment]
Maybe we update the docs page to:
* Be more similar to the polaris docs so that the names / tints align and "map" so it makes a bit more sense visually
* Click to copy-paste the names

**Comment by mihneaz** [comment]
@lloydhumphreys here's the new layout for the colors page:
![colors-page](https://user-images.githubusercontent.com/510261/51110428-ee5e5a80-17f8-11e9-8025-f0db1aac9225.png)
Abstract link: https://share.goabstract.com/0931c85f-4c48-48a5-aab8-8f212e8ddc82

Clicking on a color should trigger an Success Notification with the message: 
> *Color copied to your clipboard.*

**Comment by lloydhumphreys** [comment]
@mihneaz This is missing information about what to use for text/backgrounds - is that coming? I would be happy to work with you on this if you'd like me to. 

Otherwise this looks good to me. How about we change the titles to be those funkier names (_cerulean_ for example) rather than the all-caps-headings?

**Comment by mihneaz** [comment]
Abstract page: https://share.goabstract.com/a3374fc8-6d8e-433a-acdb-c687a4730ad6
Invisionapp Project: https://tradeshift.invisionapp.com/share/HXQ2RAVEKW7

Also invited Leo to the Invisionapp project.

**Comment by mihneaz** [comment]
@tynandebold updated files:
- Changed board titles to root color (Blue, Yellow, Orange, etc)
- Added black and white board 

![colors-page](https://user-images.githubusercontent.com/510261/51483462-613b7880-1d99-11e9-8ae5-bb3083dee50c.png)

**Comment by tynandebold** [comment]
Closed with pull #725
